### PR TITLE
Update to Quarkus 0.21.1

### PR DESCRIPTION
**Pull request by jmini**

### CODE REVIEW

File: src/main/resources/import.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Introduction to Quarkus', true, 0, null)
-INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Hibernate with Panache', false, 1, null)
-INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Visit Quarkus web site', false, 2, 'https://quarkus.io')
-INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project', false, 3, 'https://github.com/quarkusio/quarkus/')
+INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Introduction to Quarkus', true, 0, null);
```
Comment (jmini) [inline src/main/resources/import.sql:1]: As indicated by @machi1990, the `;` seems now to be mandatory.

See https://github.com/quarkusio/quarkus/issues/3417#issuecomment-518092150

Comment (jmini) [reply src/main/resources/import.sql:1]: This is also documented in the release-notes:

https://quarkus.io/blog/quarkus-0-20-0-released/#hibernate-orm-multi-line-support-in-import-scripts

**Comment by jmini** [comment]
I have updated to `0.21.1` instead.

**Comment by cescoffier** [comment]
Thanks!

**Pull request merged**